ansible.cfg

```diff
@@ -21,10 +21,6 @@ action_warnings = False
 allow_world_readable_tmpfiles = True
 stderr_callback=debug
 stdout_callback=debug
-localhost_warning = False
-
-[inventory]
-unparsed_is_failed = True
```

Comment:
@sarahotis and I talked and found this change I made in a previous PR broke some other execution patterns. Will look into this more at a later date.